tools/sigclient/cmd/sigclient.go

```diff
@@ -126,10 +127,18 @@ var functionalTestCmd = &cobra.Command{
 		maxVariableCols := 20
 		sleepDuration := 15 * time.Second
 
+		if longer {
+			totalEvents = 50_000_000
+			batchSize = 100
+			numIndices = 20
+			processCount = 10
```

Comment:
Are we not going set the process count and num indices through config?

Reply:
No, we do not want to expose these params.